services-js/access-boston/src/server/access-boston.ts

```diff
@@ -276,6 +281,76 @@ async function addGraphQl(
   });
 }
 
+async function addVelocityTemplates(server: HapiServer) {
+  server.route({
```

Comment:
This is a good candidate for generalization. You could make an array or object with the path / templatePath values in it, and iterate, calling `route` for each of them, since all 4 are exactly the same.

Just a suggestion, though, since this is dev code and therefore it's not tremendously important to remove the duplication.

Reply:
I don‘t love the duplication, but wanted to stand these up asap so the template work could be completed and delivered. I may go back later today and refactor after they get emailed out, if you think it’s a good idea/time use.

Reply:
Totally fine. It's a thing to clean up later if you're back in there or just want the exercise.